aclk/schema-wrappers/schema_wrapper_utils.h

```diff
@@ -3,6 +3,8 @@
 #ifndef SCHEMA_WRAPPER_UTILS_H
 #define SCHEMA_WRAPPER_UTILS_H
 
+#include "node_info.h"
```

Comment:
scope can be limited here to `rrd.h` AFAIK we only need RRDLABEL_SRC

Reply:
fixed it